.github/pull_request_template.md

```diff
@@ -0,0 +1,98 @@
+    * Do you think this PR requires additional review of its security implications for other reasons? **Yes** or **No**.
+
+
+### What Needs to Be Merged and Deployed Before this PR?
+
+<!--
+Add some items to the following list, or remove the entire section if it doesn't apply.
+
+Common items include:
+* Database migrations (which should always be deployed by themselves, to reduce risk).
+* New features in external dependencies (e.g. BFD).
+-->
+
+This PR cannot be either merged or deployed until the following pre-requisite changes have been fully deployed:
+
+* CMSgov/some_repo#42
+
+
+### Submitter Checklist
+
+I have gone through and verified that...:
+
+* [ ] This PR has been refactored and rebased (for help, see: [this](https://dev.to/maxwell_dev/the-git-rebase-introduction-i-wish-id-had) and [this](https://raphaelfabeni.com/git-editing-commits-part-1/), if needed, so that it is as small as it can reasonably be, in order to:
+    1. Ensure that any problems it causes have a small "blast radius".
+    2. Ensure that it'll be easier to rollback if that becomes necessary.
+    3. Ease the burden on reviewers.
+* [ ] The commits in this PR:
+    1. Reasonably explain the "what" and "why" of the changes.
+    2. Leverage JIRA's [smart commits](https://confluence.atlassian.com/jirasoftwarecloud/processing-issues-with-smart-commits-788960027.html) to reference the JIRA ticket that they're associated with. For example, aim for commit messages like this (note also [the 50/72 formatting](https://stackoverflow.com/q/2290016) used here):
+        
+        ```
+        Added the whizbang to the doodad.
+
+        The new whizbang should really simplify things for our whoosit users.
+        It was a bit tricky to get it into the doodad, but we decided that the
+        flipwhee pattern was the best approach for now. Might want to revisit
+        that in the future if it ends up being too hard to maintain.
+
+        SOMEPROJECT-42
+        ```
+        
+* [ ] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
+* [ ] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
+* [ ] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
+* [ ] Reviews are requested from both:
+    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
+    * Any relevant engineers on other projects (e.g. BFD, SLS, etc.).
+* [ ] Any deviations from the other policies in the [DASG Engineering Standards](../policies/engineering_standards.md) are specifically called out in this PR, above.
```

Comment:
Should we make this path absolute so that when used outside of this repo the link resolves?

Reply:
Excellent idea -- done!